composer.json

```diff
@@ -46,6 +47,7 @@
         "preferred-install": "dist"
     },
     "extra": {
+        "contao-manager-plugin": "Contao\\FaqBundle\\ContaoManager\\Plugin",
```

Comment:
Absolutely, but we can't enforce that. And `contao-bundle` exists since Contao 4, the manager (obviously) does not. But that sure should be best practice.